be2-scala/src/main/scala/ch/epfl/pop/model/network/JsonRpcRequest.scala

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.pop.model.network
+import ch.epfl.pop.model.network.method.Params
+import ch.epfl.pop.model.network.method.message.data.MessageData
+
+case class JsonRpcRequest(
+                           jsonrpc: String,
+                           method: Method.Method,
```

Comment:
`Method.Method` looks odd. I understand why it was done like this, but I'm wondering if it would be more understandable by renaming it `Method.MethodType`

Reply:
Answering messages even though we talked about them in today's meeting just in case someone else is following this PR :) 

Renamed it! :)